### DISCUSSION TITLE
Sonoff POW R2 not displaying voltage when load is off

### DISCUSSION THREAD

**Discussion by sunishnet** [arendst/Tasmota#15274] [Q&A] [answered]
Is there any option to view the mains voltage whenever sonoff POW R2 is powered even if switch/load is off.

Ewelink software displays input voltage even if load is off.

**Comment by barbudor** [comment, marked as answer]
https://tasmota.github.io/docs/Commands/#setoption21

**Reply by sunishnet** [reply]
Thanks, it worked. I feel that option should be the default.

**Comment by coldfire10** [comment]
Hi, I have thesame problem in simular module Sonoff POW Elite 20A (POWR320D) I set SetOption21 but there is no effect... Display only 1V-8V when relay is OFF when relay is ON working correctly.

**Reply by sfromis** [reply]
That's no "problem", but how the hardware works.

**Reply by coldfire10** [reply]
Are you sure the hardware measuring voltage is only behing the relay? This significantly limits the functionality of the module...

**Reply by arendst** [reply]
Yes. For some.

**Reply by coldfire10** [reply]
It's not good for me... Can you tell me which sonoff modules provide this functionality. All new version R3 with display don't provide this functionality yes?

**Reply by sfromis** [reply]
From devices I own, my experience is that `SetOption21 1` works at least for Sonoff Pow R2 and Sonoff Dual R3, but not Sonoff POWR316D.
